focus_converter_base/focus_converter/conversion_configs/aws/sku_price_id_S001.yaml

```
@@ -0,0 +1,8 @@
+plan_name: convert Pricing/RateCode, Pricing/RateId to SKUPriceId
```

```suggestion
plan_name: convert Pricing/RateCode, Pricing/RateId to SkuPriceId
```

according to [spec](https://github.com/FinOps-Open-Cost-and-Usage-Spec/FOCUS_Spec/blob/working_draft/specification/columns/skupriceid.md)

---

Fixed